### ISSUE TITLE
Waiting for a long time on each pip install

### ISSUE THREAD

**Issue by satyenataagyaa** [udacity/aitnd-issues#83]
The installation of requirements in the exercises for Lesson 27 doesn't seem to stick for me. I have to reinstall every time I begin an exercise, and make me wait for a long time before I can begin.

**Comment by e7dud7e** [comment]
Hi! thanks for letting us know.  Can you gives your email that you use with your Udacity account?  I can follow up with the engineering team about this.

**Comment by satyenataagyaa** [comment]
My email is: satyen@aagyaa.com. However, I usually authenticate with Google
when signing in to my Udacity classroom.

I faced this issue on all notebooks after Project 3. It consumed far too
much of my time. I submitted Project 4 this morning, and the review came
back positive, so I am no longer in need of any action toward the notebooks
in Part 1. If you are fixing it to avoid the same issue in Part 2, feel
free to ask me any questions, and/or to try any solutions. Thanks!
